src/index.html

```diff
@@ -35,9 +35,20 @@
   <div id="auth"></div>
 
   <div id="home" class="hide">
-    <div id="diary" class="btn btn-primary">Diary</div>
-    <div id="events" class="btn btn-primary">Events</div>
-    <div id="news" class="btn btn-primary">News</div>
+    <button id="diary-button" class="btn btn-primary">Diary</button>
```

Comment:
this div is being used by someone else.  They are probably expecting an id of diary.  put it back to what it was

Reply:
fixed

Reply:
done

Reply:
done